Watchman.Discord/Watchman.Discord.csproj

```diff
@@ -29,9 +29,6 @@
     <None Update="config.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Update="version.txt">
```

Comment:
czemu to usunąłeś?
chociaż hmm
możliwe że to niepotrzebne, że ta informacja powinna siedzieć w bazie a nie w formie pliku

Reply:
Aktualnie wyrzuciłem zupełnie version, kiedy bot już jest hostowany na dockerze, to już nie jest to za bardzo potrzebne, bo zawsze będzie najnowsza wersja na masterze